sql/core/src/test/resources/sql-tests/inputs/csv-functions.sql

```diff
@@ -0,0 +1,12 @@
+-- from_csv
+describe function from_csv;
+describe function extended from_csv;
+select from_csv('1', 'a INT');
```

Comment:
I think we don't need this test case since the below one tests it.

Reply:
I removed it.